src/commands/help.ts

```diff
@@ -58,15 +62,21 @@ export const command: NeedleCommand = {
 	},
 };
 
-async function getCommandDetailsEmbed(commandName: string): Promise<MessageEmbed[]> {
+async function getCommandDetailsEmbed(
+	commandName: string
+): Promise<MessageEmbed[]> {
 	const cmd = getCommand(commandName);
-	if (!cmd) { return []; }
+	if (!cmd) {
+		return [];
+	}
```

Comment:
```suggestion
	if (!cmd) return [];
```

Reply:
This shouldn't have happened, I think it's a conflict between ESLint and Prettier. I'll look into it tomorrow.